src/components/board/lcd/style.css

```diff
@@ -0,0 +1,25 @@
+.lcd-container {
+  border: 1px solid lightgrey;
```

Comment:
Maybe the border could be made either darker or thicker. For me, it was hard to recognize the LCD display on the board at first because in contrast to the other elements on the board, it is empty by default.

Reply:
made it thicker and changed to black (kind of like on real board)